deploy/firesim

```diff
@@ -257,7 +263,15 @@ def main(args):
 
     # tasks that have a special config/dispatch setup
     if args.task == 'managerinit':
-        managerinit()
+        if args.copyonly:
+            managerinit()
+        else:
+            rootLogger.warning("DEPRECATION: managerinit will move AWS setup strictly to awsinit in a future release.")
+            awsinit()
+            managerinit()
```

Comment:
Given that managerinit warns the user to run awsinit, I think we should just bite the bullet, update the docs and not introduce a new option `-o` for something that is being deprecated as the option is being added.